### ISSUE TITLE
There seems to be no way to change the process data

### ISSUE THREAD

**Issue by gralpli** [matwey/SOEM-rs#8]
`Context` has a method `.slaves()` which returns an immutable `&'a [Slave]`, but I need a `&mut Slave` to be able to call `.outputs(&mut self)` on it.

```rust
let inputs = context.slaves()[0].inputs(); // does work
…
let outputs = context.slaves()[0].outputs(); // does not work
```

The context has also captured a mutable reference to the IO map, so I can’t access that either.

Is there actually no way of changing the output data, or am I missing something?

**Comment by gralpli** [comment]
✨ Ladies and gentlemen! You can see right here a perfect demonstration of rubber duck problem solving! ✨

I’ve used the version from crates.io and just after opening this issue, I’ve seen that the last commit of the master branch contains a fix for this.

**Comment by matwey** [comment]
Hi @gralpli 

It seems that @flosse solved your issue in #7. My bad, I had to create new version.

**Comment by matwey** [comment]
I have created `0.3.0` tag. It should take some time for CI to publish new crate.
